packages/react-core/src/components/ExpandableSection/ExpandableSection.tsx

```diff
@@ -61,7 +81,9 @@ export class ExpandableSection extends React.Component<ExpandableSectionProps, E
     displaySize: 'default',
     isWidthLimited: false,
     isIndented: false,
-    contentId: ''
+    contentId: '',
+    variant: 'default',
+    truncateMaxLines: 3
```

Comment:
Given what @mcoker said about core already having a default of 3 fir the truncation, I think this should be undefined.  we should only `setLineClamp`.